langmeta/io/js/src/main/scala/java/io/File.scala

```diff
@@ -44,6 +44,10 @@ class File(path: String) {
 }
 
 object File {
+  def listRoots() = 
```

Comment:
Try to add an explicit `: Array[File]` return type, it's probably getting inferred to `List[File]` which produces a different jvm signature that fails to link. The reason it still compiles fine is because the JVM boot classpath is available during compilation, but not linking.